### PR TITLE
Added method to export tags

### PR DESCRIPTION
**Pull request by SreeHarshaNelaturu**

### CODE REVIEW

File: remo/domain/dataset.py [hunk 1]
```diff
@@ -260,6 +262,7 @@ def export_annotations(
                 annotation_format=annotation_format,
                 export_coordinates=export_coordinates,
                 full_path=full_path,
+                export_tags=str(export_tags).lower()
```
Comment (vmanilo) [inline remo/domain/dataset.py:265]: @SreeHarshaNelaturu here you can pass `export_tags` value as it is

```suggestion
                export_tags=export_tags
```

Comment (SreeHarshaNelaturu) [reply remo/domain/dataset.py:265]: Its a boolean value, is that okay?

File: remo/domain/dataset.py [hunk 2]
```diff
@@ -290,6 +295,7 @@ def export_annotations_to_file(
                 annotation_format=annotation_format,
                 full_path=full_path,
                 export_coordinates=export_coordinates,
+                export_tags=str(export_tags).lower()
```
Comment (vmanilo) [inline remo/domain/dataset.py:298]:
```suggestion
                export_tags=export_tags
```

File: remo/sdk.py [hunk 3]
```diff
@@ -413,6 +414,7 @@ def export_annotations(
             annotation_format=annotation_format,
             export_coordinates=export_coordinates,
             full_path=full_path,
+            export_tags=str(export_tags).lower()
```
Comment (vmanilo) [inline remo/sdk.py:417]:
```suggestion
            export_tags=export_tags
```

File: remo/sdk.py [hunk 4]
```diff
@@ -432,12 +435,14 @@ def export_annotations_to_file(
...
             full_path: uses full image path (e.g. local path), can be one of ['true', 'false'], default='false'
             export_coordinates: converts output values to percentage or pixels, can be one of ['pixel', 'percent'], default='pixel'
+            export_tags: exports the tags to a CSV file, it can be one of [True, False], default=True
         """
         content = self.export_annotations(
             annotation_set_id,
             annotation_format=annotation_format,
             export_coordinates=export_coordinates,
             full_path=full_path,
+            export_tags=str(export_tags).lower()
```
Comment (vmanilo) [inline remo/sdk.py:445]:
```suggestion
            export_tags=export_tags
```

File: remo/sdk.py [hunk 5]
```diff
@@ -394,7 +394,8 @@ def export_annotations(
         annotation_set_id: int,
         annotation_format: str = 'json',
         export_coordinates: str = 'pixel',
-        full_path: str = 'true',
+        full_path: bool = True,
```
Comment (vmanilo) [inline remo/sdk.py:397]: @SreeHarshaNelaturu if you to change `full_path` as well, you would need also convert bool to str in api.py, and check other places where `full_path` parameter uses

File: remo/api.py [hunk 6]
```diff
@@ -358,25 +358,26 @@ def get_annotation_set(self, id):
         return self.get(url).json()
 
     def export_annotations(
-        self, annotation_set_id: int, annotation_format='json', export_coordinates='pixel', full_path='true'
+        self, annotation_set_id: int, annotation_format='json', export_coordinates='pixel', full_path='true', export_tags: bool = True,
```
Comment (vmanilo) [inline remo/api.py:361]:
```suggestion
        self, annotation_set_id: int, annotation_format='json', export_coordinates='pixel', full_path=True, export_tags: bool = True,
```

File: remo/domain/annotation_set.py [hunk 7]
```diff
@@ -115,16 +115,16 @@ def add_image_annotation(self, image_id: int, annotation: Annotation):
         self.sdk.add_annotation(self.id, image_id, annotation)
 
     def export_annotations(
-        self, annotation_format: str = 'json', export_coordinates: str = 'pixel', full_path: str = 'true'
+        self, annotation_format: str = 'json', export_coordinates: str = 'pixel', full_path: bool = True, export_tags: book = True
```
Comment (vmanilo) [inline remo/domain/annotation_set.py:118]:
```suggestion
        self, annotation_format: str = 'json', export_coordinates: str = 'pixel', full_path: bool = True, export_tags: bool = True
```

**Pull request merged**